docs/pnpm-lock.yaml

```diff
@@ -0,0 +1,10284 @@
+lockfileVersion: '9.0'
```

Comment:
Could you also maybe give us some context as to why we have this lock file besides the package lock file?

Reply:
I am using a new package manager
PNPM instead of NPM
this makes upgrading way easier (for me)
normally this could result in some issues, since it can change some versions of packages you don't intent.
But since I am upgrading all packages anyway, this issue won't be present. :)

Reply:
Ok, great, so could older package files then be deleted?